src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -58,15 +61,21 @@
         }) }}
       {% endblock %}
 
-      {% block product_tab_stock %}
-        <div role="tabpanel" class="form-contenttab tab-pane" id="stock-tab">
-          {{ form_widget(productForm.stock) }}
-        </div>
-      {% endblock %}
-
-      {% block product_tab_combinations %}
-        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig') }}
-      {% endblock %}
+      {% if productType == 'virtual' %}
+        {% block product_tab_virtual %}
+          {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/virtual.html.twig') }}
+        {% endblock %}
+      {% elseif productType == 'combinations' %}
+        {% block product_tab_combinations %}
+          {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Tabs/combinations.html.twig') }}
+        {% endblock %}
+      {% else %}
+        {% block product_tab_stock %}
```

Comment:
I guess this needs to be discussed with @MatShir but I'm not sure stock tab needs to be removed for virtual product. It seems to be only needed for Combinations product in the legacy page

Reply:
well just check legacy page. Depending on product type the tabs are changing (its either stock, combination, virtual product or pack product tab) it only changed a bit how we handle it in code, but not in UX perspective. If the tab is Virtual product, it doesn't mean that we will not be able to use some fields from stockType.

Reply:
Yeah I didn't get it correctly at first, the stock form is in virtual tab, or it's displayed alone It does work but as I said in other comments I'm not completely convinced yet with this splitting 🤔  I would keep a single tab content template for virtual + stock but with a single Tab header link which only name is changed